README.md

```diff
@@ -1,86 +1,49 @@
-[Cozy](https://cozy.io) Drive for Desktop <sup>(beta)</sup>
-===========================================================
+# [Cozy](https://cozy.io) Drive for Desktop <sup>(beta)</sup>
+
+[![Cozy Logo](https://raw.github.com/cozy/cozy-setup/gh-pages/assets/images/happycloud.png)](https://cozy.io)
 
 [![Latest release](https://img.shields.io/github/release/cozy-labs/cozy-desktop/all.svg)](https://github.com/cozy-labs/cozy-desktop/releases)
 [![Licence](https://img.shields.io/github/license/cozy/cozy-drive.svg)](https://github.com/cozy/cozy-drive/blob/master/LICENSE)
 <br>
 [![Windows build](https://img.shields.io/appveyor/ci/cozy/cozy-desktop/master.svg?label=windows)](https://ci.appveyor.com/project/cozy/cozy-desktop/branch/master)
-[![Linux & macOS build]( 	https://img.shields.io/travis/cozy-labs/cozy-desktop/master.svg?label=mac/linux)](https://travis-ci.org/cozy-labs/cozy-desktop/branches)
-[![Code climate](https://img.shields.io/codeclimate/github/cozy-labs/cozy-desktop.svg)](https://codeclimate.com/github/cozy-labs/cozy-desktop)
+[![Linux & macOS build](https://img.shields.io/travis/cozy-labs/cozy-desktop/master.svg?label=mac/linux)](https://travis-ci.org/cozy-labs/cozy-desktop/branches)
+[![Code climate](https://img.shields.io/codeclimate/maintainability-percentage/cozy-labs/cozy-desktop.svg)](https://codeclimate.com/github/cozy-labs/cozy-desktop)
 [![Coverage](https://img.shields.io/codecov/c/github/cozy-labs/cozy-desktop/master.svg?label=coverage)](https://codecov.io/gh/cozy-labs/cozy-desktop/list/master)
 
-## What is Cozy?
+Cozy Drive for Desktop allows you to synchronize the files stored in [your Cozy](https://cozy.io) with your laptop and/or desktop computer.
+It replicates your files on your hard drive and applies the changes you make to them on other synchronized devices and on [your online Cozy](https://github.com/cozy/cozy-stack).
```

Comment:
Why link to the cozy-stack repo and not to the Cozy web repo as for the first sentence?